feature/home/presentation/viewmodel/src/main/java/com/chocolatecake/viewmodel/people/mapper/PeopleDataUiMapper.kt

```diff
@@ -0,0 +1,23 @@
+package com.chocolatecake.viewmodel.people.mapper
+
+import com.chocolatecake.entities.PeopleDataEntity
+import com.chocolatecake.mapper.Mapper
+import com.chocolatecake.viewmodel.people.PeopleDetailsUiState
+import javax.inject.Inject
+
+class PeopleDataUiMapper @Inject constructor() :
+    Mapper<PeopleDataEntity, PeopleDetailsUiState.PeopleDataUiState> {
```

Comment:
should map to PeopleDetailsUiState and not PeopleDetailsUiState.PeopleDataUiState